### DISCUSSION TITLE
I didn't get 0.25 as my farm.

### DISCUSSION THREAD

**Discussion by tongthb** [Chia-Network/chia-blockchain#17683] [Support]
Hello Admin
I stay to mining by Self Pooling and last 3 block to I found I didn't get 0.25 (5,021,676; 5,001,345; 4,698,039).
It addresses as below
0x02e87fd5860c281be6351c1ce3ff50e86703b3a91c6135074b73bc4743421705
xch1jt35v4fgtk0hwcyqlrepqn06z9450vyay7q3xnpk5g92unhdga5ssp9uem
Best Regards
Worawit
![Chia](https://github.com/Chia-Network/chia-blockchain/assets/52958210/887cd84d-15ce-435a-b1d0-3b6d5a7c8a84)
![chia0](https://github.com/Chia-Network/chia-blockchain/assets/52958210/a296216d-dad2-4614-bd58-0ad214a6805b)

**Comment by BrandtH22** [comment]
Hey @tongthb , as a heads up we can generally provide more timely and thorough support on our discord server (https://discord.gg/chia).

The first thing I would do is verify the payout settings in the GUIs farm section settings and ensure it is set to the noted xch address.
The other thing is if you are farming gigahorse compressed plots, a portion of the farmer rewards are sent to max as a developer fee so that could also be the case.

**Reply by tongthb** [reply]
I checked and found 0.25 is on this wallet 
xch1tatcd6lxu6edxvn9gvyryt2dv3xla07wuwepqmpw2huans3f6ekqc6l8qx
The wallet is in the Chia config as per the attached photo. I don't know why my wallet did change.
![Chia 01](https://github.com/Chia-Network/chia-blockchain/assets/52958210/48e8aaf7-c3dc-4296-b73f-a7c2a6c1f887)
![Chia02](https://github.com/Chia-Network/chia-blockchain/assets/52958210/94e27a92-fe78-440f-af59-c6199f4070c5)

**Reply by BrandtH22** [reply]
Hey @tongthb , when a new config is created it will use the wallets first address as the payout instructions so as to ensure that a keyset on the machine has access to the funds. 

I know when I was swapping farming machines at one point I had created a new keyset accidentally before importing my farming keyset so my payout instructions were set to that new keyset as it was the first one created on the new machine. 
I am not sure if that is the case here but if you do have another keyset on that machine I would sync it to see if that is where the funds are stored.

Let us know what other questions arise and whether we can help dive deeper into identifying the keyset associated with that xch address 👍 (we can use some of the chia client tools to verify which addresses are owned by your keyset)

**Reply by tongthb** [reply]
Plese can you check this wallet for identifying the keyset associated.

xch1tatcd6lxu6edxvn9gvyryt2dv3xla07wuwepqmpw2huans3f6ekqc6l8qx

Thank you very much for your answer.

**Reply by BrandtH22** [reply]
Hey @tongthb , the best way to identify whether this address belongs to one of your keysets is by running the chia keys derive command with the search flag. 

If you are in mac or linux you can use terminal , if using windows then I recommend powershell. 

You can run this command to determine if that xch address belongs to one of your mnemonics:
`chia keys derive search -t xch1tatcd6lxu6edxvn9gvyryt2dv3xla07wuwepqmpw2huans3f6ekqc6l8qx -d`

As always we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

**Reply by tongthb** [reply]
I am apologize, this issue was fixed because the second wallet is mine thank you very much for your answer☺️🙏
